core/scripts/cli/wallet/createNewAccount.js

```diff
@@ -0,0 +1,26 @@
+const inquirer = require("inquirer");
+const style = require("../textStyle");
+
+module.exports = async web3 => {
+  const confirm = await inquirer.prompt({
+    type: "confirm",
+    name: "createNewAccount",
+    message: style.bgYellow(
+      `We will never store your private keys, so please ensure that your computer is secure and you keep it a secret. Anybody who knows your private key controls your account! Type 'y' to confirm your understanding.`
```

Comment:
I like this a lot -- one TODO you might want to add (definitely for a later PR) is to give the user the option to generate a mnemonic using the [bip39 package](https://www.npmjs.com/package/bip39).

Reply:
done check out #867 

and also check out #868 for other ideas on making it easier to create **usable** accounts for the user

Reply:
Thanks for being so disciplined about creating issues for all this stuff -- that's super helpful.